metrics/prometheus.go

```diff
@@ -415,6 +415,32 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.PidsMetrics) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:      "container_pids_max",
+				help:      "The number of container max pids, infinity if value is zero",
```

Comment:
reword:
The maximum number of processes allowed in the container, infinity if value is zero

Reply:
I will fix it

Reply:
I have fixed it